.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build-macos:
+    name: Build macOS
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Add targets
```

Comment:
`actions-rs/toolchain` doesn't currently support multiple targets via action inputs, so we have to do this manually (see actions-rs/toolchain#62).